test/modules/core/utils/apply-styles.spec.ts

```diff
@@ -0,0 +1,30 @@
+/* global document */
+import test from 'tape-promise/tape';
+import {applyStyles} from '@deck.gl/core/utils/apply-styles';
+
+const TEST_CASES = [
+  {
+    title: 'CSS variable',
+    argument: {property: '--my-var', value: 'red'},
+    result: {property: '--my-var', value: 'red'}
+  },
+  {
+    title: 'camelCase property',
```

Comment:
Can you please test the faulty Kebab-case as well? I wonder what our runner will do